rockets/ws/types.h

```diff
@@ -51,7 +51,21 @@ enum class Recipient
 };
 
 /**
- * A response followed after an incoming message/request.
+ * A request from a client during handleText()/handleBinary().
+ */
+struct Request
+{
+    Request(std::string message_, const uintptr_t clientID_ = 0)
```

Comment:
I am not sure if this constructor is needed, the default {} initialization list constructor should work already.
If it is, be careful about taking message_ by value; I think it has to be std::moved into message otherwise a copy occurs.